extlinks/organisations/templates/organisations/organisation_charts_include.html

```diff
@@ -166,8 +187,264 @@ <h3>Latest link events</h3>
     evt.currentTarget.className += " active";
   }
 
-  // Click all Link Events buttons
-  {% for collection_name, collection in collections.items %}
-    document.getElementById('{{ collection_name }}_linkEvents_button').click();
-  {% endfor %}
+  function getLinksCount(collection_id, form_data){
+    $.ajax({
+      url: "{% url 'organisations:links_count' %}?collection=" + collection_id + "&form_data=" + JSON.stringify(form_data),
```

Comment:
Another tiny nit, feel free to ignore: 
Can we use[ template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here instead of string concat to build the URL?